client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-methods/utils.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import formatCurrency from 'lib/format-currency';
+
+export const getMethodSummary = ( method, currency ) => {
+	switch ( method.methodType ) {
+		case 'flat_rate':
+			return translate( 'Cost: %(cost)s', { args: {
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Cost: %s' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).